blockchains/avalanche/info/info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Avalanche",
+    "website": "http://avax.network",
+    "source_code": "https://github.com/ava-labs",
+    "white_paper": "https://www.avalabs.org/whitepapers",
+    "description": "Avalanche is an open-source platform for launching highly decentralized applications, new financial primitives, and new interoperable blockchains.",
+    "explorer": "https://explorer.avax.network",
+    "research": "https://research.binance.com/en/projects/avalanche",
+    "symbol": "AVAX",
+    "type": "coin",
+    "decimals": 9,
+    "status": "active",
+    "tags": [
+        "staking-native"
+    ]
+}
```

Comment:
For the logo can you use just a symbol, similar to CMC: https://coinmarketcap.com/currencies/avalanche/

Reply:
Can do! Logo updated.